backend/src/models/item/postgres.js

```diff
@@ -96,11 +138,20 @@ const ItemRepo = (postgres) => {
     }
   };
 
-  // Gets all bids for a specified item id
+  /**
+   * Gets all bids for a specified item id
+   * 
+   * @type {string}
+   */
   const getBidsForItemSQL = `
     SELECT * from bids where item_id=$1;`;
 
-  // Given an item id, get all bids
+  /**
+   * Given an item id, get all bids
+   * 
+   * @param  {number} item_id - ID of item to get bids for
+   * @return {Array<{0: Bid, 1: String}>} - Array with Rafflebay Bid Object and error (only one or the other)
```

Comment:
Technically the return type is an array with index 0 having an array of bids cause its multiple.

So Array<{0: Array<Bid>, 1: String}>}